static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -708,12 +708,20 @@ svg.main-menu__icon--social__svg {
 
 .tertiary-navigation__item {
     width: calc(50% - #{$gs-gutter / 2});
+    height: $gs-row-height;
     border-top: 1px solid $neutral-5;
+    padding-top: 2px;
     padding-bottom: $gs-baseline / 2;
 }
 
+.tertiary-navigation__text {
+    width: calc(100% - #{$gs-gutter});
+    float: left;
+}
+
 .tertiary-navigation__link {
     @include fs-textSans(5);
+    line-height: 18px;
```

Comment:
Minor, but could this be a proportion of the item height (i.e - you expect the item to be two lines so the line height is `$gs-row-height / 2`